package.json

```diff
@@ -36,7 +36,6 @@
     "@uppy/progress-bar": "^3.0.0",
     "@uppy/tus": "^3.0.1",
     "a11y-datepicker": " ^0.9.0",
-    "axios": "^0.27.2",
```

Comment:
Shoudn't there be an updated `yarn.lock`, too?

Reply:
true 14152f743723a784ecdd18b67422106eb25e1969